src/Orleans.TestingHost/AppDomainSiloHost.cs

```diff
@@ -56,13 +50,6 @@ public void Shutdown()
         {
             this.host.StopAsync().GetAwaiter().GetResult();
         }
-
-        private void InitializeTestHooksSystemTarget()
```

Comment:
where is this part of logic moved to ?

Reply:
Moved to [TestClusterHostFactory](https://github.com/dotnet/orleans/pull/3878/files#diff-238a7057e8b8da8eeefdc924dd6abc7cR254)